### PR TITLE
Writing Tests for PHPUnit: various improvements

### PR DESCRIPTION
**Pull request by lex111**

### CODE REVIEW

File: src/writing-tests-for-phpunit.rst [hunk 1]
```diff
@@ -1070,6 +1070,6 @@ functions on arrays or objects.
 
 In this example the difference in the first index between
 ``1`` and ``'1'`` is
-reported even though assertEquals considers the values as a match.
+reported even though ``assertEquals()`` method considers the values as a match.
```
Comment (keradus) [inline src/writing-tests-for-phpunit.rst]: could be worth to mention `assertSame()` here

**Comment by lex111** [comment]
ping @sebastianbergmann

**Comment by lex111** [comment]
Hi @sebastianbergmann how are you? What is the problem with the adoption of this PR?

File: src/writing-tests-for-phpunit.rst (hunk 1 above)
Comment (keradus) [reply src/writing-tests-for-phpunit.rst]: ping @lex111

Comment (lex111) [reply src/writing-tests-for-phpunit.rst]: I saw, but my changes only affect the external appearance and just change the names, not the content itself, so I do not know what to write this about that :(

Comment (keradus) [reply src/writing-tests-for-phpunit.rst]: `To have them not matching each other, use ``assertSame()``, which doesn't check equality but identity`
or sth like that, perhaps ?

**Pull request merged**